app/bingey-ui/src/utils/api.js

```diff
@@ -0,0 +1,24 @@
+import axios from 'axios';
+
+const BASE_URL = 'http://localhost:3001';
+
+const formatStringForSearch = (value) => {
+  return value.toLowerCase().replace(' ', '+');
+};
+
+export const search = async (queryString) => {
+  queryString = formatStringForSearch(queryString);
+  try {
+    return (await axios.get(`${BASE_URL}/search?q=${queryString}`)).data;
+  } catch (error) {
```

Comment:
I'm not sure if we should have try/catch blocks here, is there any important information they can capture here? Also maybe we should export at the end of the file like: `export default {search, createTitle}`, I think it would minimize the code instead of writing export in front of every function.

Reply:
Yeah, you're right, it's easier to find the exports this way